graviti/dataset/dataset.py

```diff
@@ -103,14 +110,32 @@ def factory_getter(
                 128,
                 factory_getter,
             )
-            # catalog = get_catalog(
-            # self._url,
-            # self._access_key,
-            # self._dataset_id,
-            # draft_number=self._status.draft_number,
-            # commit=self._status.commit_id,
-            # )
-            self._data[sheet_name] = DataFrame.from_lazy_factory(factory)
+            catalog = get_catalog(
+                self._url,
+                self._access_key,
+                self._dataset_id,
+                commit=self._commit_id,
+            )
+            first_data_details = data_details["dataDetails"][0]
+            remote_data = RemoteData.from_response_body(
+                first_data_details,
+                url=URL(first_data_details["url"], lambda: "update is not supported currently"),
+            )
+
+            notes = get_notes(
+                self._url,
+                self._access_key,
+                self._dataset_id,
+                commit=self._commit_id,
+            )
+
+            schema = yaml.load(
+                catalog_to_schema(Catalog.loads(catalog), remote_data, Notes.loads(notes)),
+                yaml.Loader,
+            )
+            extractors = get_extractors(schema)
+            lazylists = factory.get_lazylists(extractors)
```

Comment:
Save the variable `extractors`?

Reply:
like this `factory.get_lazylists(get_extractors(schema))`?

Reply:
Yes.